post-deployment/openstack/net2-ingress/main.yml

```
@@ -0,0 +1,112 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars:
+    namespace: "net2-ingress"
+    net2NetworkName: "{{ net2ExternalNetwork | lower }}"
```

If spaces are replaced in the deploy repo... 

```suggestion
    net2NetworkName: "{{ net2ExternalNetwork | replace(' ','-') | lower }}"
```